app/models/user.rb

```diff
@@ -1,8 +1,9 @@
 class User < ApplicationRecord
+  include DeviseTokenAuth::Concerns::User
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable
+         :rememberable, :validatable, :confirmable
```

Comment:
I accept `:rememberable` but `:confirmable` I can not. Because if you add this fields, you assume devise will send a mail to confirm the address at sign_up. But here you do not use sing_up, and we still not handle mail.

Reply:
Ok, I'm getting rid of :confirmable